src/api/app/models/bs_request.rb

```diff
@@ -345,14 +345,14 @@ def obsolete_reviews(opts)
           review.by_package && review.by_package == opts[:by_package]
         logger.debug "Obsoleting review #{review.id}"
         review.state = :obsoleted
-        review.save
+        review.save!
         history = HistoryElement::ReviewObsoleted
-        history.create(review: review, comment: "reviewer got removed", user_id: User.current.id)
+        history.create!(review: review, comment: "reviewer got removed", user_id: User.current.id)
 
         # Maybe this will turn the request into an approved state?
         if state == :review && reviews.where(state: "new").none?
           self.state = :new
-          save
+          save!
```

Comment:
@Ana06, this `save!` method is not a common `save!` of rails, it has been overridden in line 64